test/compilable/interpret3.d

```diff
@@ -7731,3 +7731,14 @@ bool foo17407()
 
 static assert(!foo17407);
 
+/**************************************************/
+// https://issues.dlang.org/show_bug.cgi?id=18057
+// Recursive field initializer causes segfault.
+
+struct RBNode(T)
+{
+    RBNode!T *copy = new RBNode!T;
+}
+
+static assert(!__traits(compiles, { alias bug18057 = RBNode!int; }));
```

Comment:
Shouldn't we test for the exact error message here?

Reply:
If you insist.  Adding more files to the testsuite has been complained about in the past, as it is gradually slowing down the time it takes to run the testsuite vs. if all tests were kept in one mega file.  (Not saying everything should go in one mega file, but logically grouped may be a start :-)

Reply:
I created an issue to note this, incase anyone wants to take the baton: https://issues.dlang.org/show_bug.cgi?id=18235

Reply:
The exact error message only needs to be tested once by the test suite, the others can use `__traits(compiles, ... )`. So, the coverage results can be used to determine if the exact message is ever tested or not.

Reply:
I just moved it to `fail_compilation`. It's just that adding `static assert(!__traits(compiles,` to `compilable` tests strikes me as an anti-pattern